src/Gateway/Events/PresenceUpdate.php

```diff
@@ -19,7 +20,7 @@ class PresenceUpdate
     public string $status;
 
     /**
-     * @var \Ragnarok\Fenrir\Parts\Activity[]
+     * @var Activity[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)